src/bindings/python/tests/test_runtime/test_properties.py

```diff
@@ -274,7 +274,7 @@ def test_properties_hint_model():
 
 
 @pytest.mark.skipif(os.environ.get("TEST_DEVICE", "CPU") != "CPU", reason=f"Cannot run test on device {os.environ.get('TEST_DEVICE')}, Plugin specific test")
-def test_single_property_setting():
+def test_single_cpu_property_setting():
     core = Core()
```

Comment:
I suppose this test can be generic. Moreover, number of streams is supported by most (if not all) plugins. 
So, probably, we just need to remove limitations from original version of this test

Reply:
updated

Reply:
but we still have skip here..